src/Microsoft.ML.Core/Microsoft.ML.Core.csproj

```diff
@@ -12,6 +12,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="System.ComponentModel.Composition" Version="4.5.0" />
```

Comment:
>="4.5.0" [](start = 73, length = 8)

Probably want to make this version in the more global file (same place `SystemMemoryVersion` we see below is set).